### DISCUSSION TITLE
Impossible to select 'skip day' (or 'failed task') with 'hide completed' option - solution discussion.

### DISCUSSION THREAD

**Discussion by lukaszlew** [iSoron/uhabits#1053] [Feature Requests]
With 'hide completed', long press marks the task as done and there is no option to mark it as 'skip day' or 'failed'.
My list is over 60 habits long and it is hard to disable 'hide completed', find it, and re-enable it.

I was thinking about it for few days now and there are 2 ideas:

1. Holding the button shows a drop-down menu with all the possible options. 
'OK' is the default highlighted option and it will be chosen if you just lift your finger.

2. The alternative is to keep the task visible for few seconds after changing it status.

**Comment by NomarCub** [comment]
This would be a nice fix, especially with https://github.com/iSoron/uhabits/discussions/905 completed.
Issue https://github.com/iSoron/uhabits/issues/1055 is tracking progress on a solution to this, for anyone interested.
